packages/acceptance-tests/pkg-tests-specs/sources/features/installArtifactCleanup.test.ts

```diff
@@ -4,7 +4,8 @@ const lsStore = async (path: PortablePath) => {
   return await xfs.readdirPromise(`${path}/${Filename.nodeModules}/.store` as PortablePath);
 };
 
-describe(`Install Artifact Cleanup`, () => {
+// These tests randomly fail on the CI and until we figure out why they're just noise
+describe.skip(`Install Artifact Cleanup`, () => {
   describe(`pnpm linker`, () => {
```

Comment:
For future-proofing purposes, can you move the `.skip` and the comment to the `pnpm linker` describe just in case we add artifact cleanup tests for the other linkers too?

Reply:
Good point, done 👍